spi-sample-pos-2/package.json

```
@@ -20,7 +20,18 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "prepare": "cd .. && husky install spi-sample-pos-2/.husky",
+    "check-prettier": "npx prettier --check .",
```

With this one, you can just do the following as `npx` will reach out remotely to grab a script to run
```suggestion
    "check-prettier": "prettier --check .",
```

---

Yeah sure, Matt, we will update this line 🙌